functions/index.js

```diff
@@ -0,0 +1,8 @@
+const functions = require('firebase-functions');
+
+// // Create and Deploy Your First Cloud Functions
```

Comment:
I'm actually going to leave that for the time being since I don't have any Firebase Functions set up, but we will likely be adding some in the future. This file was generated by Firebase.